plugin/editorconfig.vim

```diff
@@ -503,18 +507,20 @@ function! s:ApplyConfig(config) " {{{1
 
 " Set the indentation style according to the config values
 
-    if has_key(a:config, "indent_style")
+    if index(g:EditorConfig_disable_rules, 'indent_style') < 0 &&
+                \ has_key(a:config, "indent_style")
```

Comment:
Instead of repeating this "&&" so many times, I think it's probably better to write a function to do this job.